src/api/rate_limiter.py

```diff
@@ -0,0 +1,70 @@
+import asyncio
+import requests
+import uuid
+
+from datetime import datetime
+
+
+class RequestEntry(object) :
+    """
+    Class for request object to put in the queue for the rate limiter
+    """
+    def __init__(self,url,cookies,key,method) -> None:
+        self.url = url
+        self.cookies = cookies
+        self.key = key
+        self.method = method
+
+class RateLimiter() :
+    """
+    Class that takes care of sending the requests at a maximum rate (25/sec) 
+        and giving back the result to the calling functions when it is received (async probably)
+    """
+    def __init__(self) :
+        self.requests = {}
+        self.queue = asyncio.Queue()
+        asyncio.create_task(self.handle_requests())
+
+
+    async def handle_requests(self) :
+        print(f"Starting rate_limiter task...")
```

Comment:
You should use `logging` instead :wink:

Reply:
but maybe you're not used to it, i should introduce it in another PR if that's the case

Reply:
I don't know indeed but i can add it :wink:

Reply:
If you're brave enough to dig into it you can, otherwise no worries I'll probably change every print I see when I'll add it :wink:

Reply:
Now there is logging, a very basic one. I used logging.basicConfig meaning I use the global object logging directly which is considered not a very good habit. But as long as we don't need different loggers this can do the job very simply I think.